deploy/README.md

```diff
@@ -1,74 +0,0 @@
-# StreamPark on Docker and Kubernetes Quick Start
```

Comment:
Why remove this, has the docker README been migrated elsewhere? If migrating elsewhere, it is recommended that the path to the new documentation be declared in README.md, rather than remove it, such as,

# StreamPark For Docker

* [StreamPark Start With Docker](http://xxx)